pkg/kubelet/apis/podresources/v1alpha1/api.proto

```diff
@@ -0,0 +1,43 @@
+// To regenerate api.pb.go run hack/update-generated-pod-resources.sh
```

Comment:
Split in two or three different commits:
- gRPC API and constants
- [Optional] the server and the tests
- Generated golang code

Reply:
done.  I decided to do:

* grpc API
* generated go code + hack generating scripts
* everything else